sdk/storage/Azure.Storage.Webjobs.Extensions.Blobs/tests/Azure.WebJobs.Extensions.Storage.Blobs.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+  </PropertyGroup>
+  <PropertyGroup>
+    <Version>1.0.0-beta.1</Version>
+    <Description>This extension adds bindings for Storage</Description>
+    <IsPackable>false</IsPackable>
+    <PublicSign>false</PublicSign>
+    <DelaySign>true</DelaySign>
+    <AssemblyOriginatorKeyFile>PublicKey.snk</AssemblyOriginatorKeyFile>
```

Comment:
I think we can trim some or maybe even all of these.

Reply:
I gave it a try. Net FX isn't happy about these not being present. It doesn't like TestCommon signature for some reason. We'll revisit that when we get rid of TestCommon.